EMPRESS/ass_6_calc_class.py

```diff
@@ -0,0 +1,43 @@
+
+a = 2
+b = 10
```

Comment:
Defining these variables at the top of your file is fine but they should be capitalized to say they are constants. Constant variables are traditionally put at the top of files and are written in all caps with underscores such as this:

IM_A_CONSTANT_VARIABLE = 'I live at the top of the file and shoudnt be changed in the rest of the code'